egs/lre07/v2/run.sh

```diff
@@ -0,0 +1,238 @@
+  # warping.
+  utils/subset_data_dir.sh data/train_novtln 5000 data/train_5k_novtln
+
+  # Note, we're using the speaker-id version of the train_diag_ubm.sh script, which
+  # uses double-delta instead of SDC features to train a 256-Gaussian UBM.
+  sid/train_diag_ubm.sh --nj 12 --cmd "$train_cmd" data/train_5k_novtln 256 \
+    exp/diag_ubm_vtln
+  lid/train_lvtln_model.sh --mfcc-config conf/mfcc_vtln.conf --nj 12 --cmd "$train_cmd" \
+     data/train_5k_novtln exp/diag_ubm_vtln exp/vtln
+
+  for t in lre07 train; do
+    lid/get_vtln_warps.sh --nj 12 --cmd "$train_cmd" \
+       data/${t}_novtln exp/vtln exp/${t}_warps
+    cp exp/${t}_warps/utt2warp data/$t/
+  done
+fi
+
+
+
+#Uncomment the following part if use_vtln=true above
+#utils/fix_data_dir.sh data/train
+#utils/filter_scp.pl data/train/utt2warp data/train/utt2spk > data/train/utt2spk_tmp
+#cp data/train/utt2spk_tmp data/train/utt2spk
+#utils/fix_data_dir.sh data/train
+
+
+cp -r data/train data/train_dnn
+cp -r data/lre07 data/lre07_dnn
+
+
+# Extract language recognition features
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 12 --cmd "$train_cmd" \
+  data/train exp/make_mfcc $mfccdir
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc.conf --nj 12 --cmd "$train_cmd" \
+  data/lre07 exp/make_mfcc $mfccdir
+
+# Extract DNN features
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc_hires.conf --nj 12 --cmd "$train_cmd" \
+    data/train_dnn exp/make_mfcc $mfccdir
+
+steps/make_mfcc.sh --mfcc-config conf/mfcc_hires.conf --nj 12 --cmd "$train_cmd" \
+    data/lre07_dnn exp/make_mfcc $mfccdir
+
+for name in lre07_dnn train_dnn lre07 train; do
+    utils/fix_data_dir.sh data/${name}
+done
+
+lid/compute_vad_decision.sh --nj 12 --cmd "$train_cmd" data/train \
+  exp/make_vad $vaddir
+lid/compute_vad_decision.sh --nj 12 --cmd "$train_cmd" data/lre07 \
+  exp/make_vad $vaddir
+
+
+for name in train lre07; do
+  cp data/${name}/vad.scp data/${name}_dnn/vad.scp
+  cp data/${name}/utt2spk data/${name}_dnn/utt2spk
+  cp data/${name}/spk2utt data/${name}_dnn/spk2utt
+  utils/fix_data_dir.sh data/${name}
+  utils/fix_data_dir.sh data/${name}_dnn
+done
+
+# Subset training data for faster sup-GMM initialization.
+utils/subset_data_dir.sh data/train 32000 data/train_32k
+utils/fix_data_dir.sh data/train_32k
+utils/subset_data_dir.sh data/train_dnn 32000 data/train_dnn_32k
+utils/fix_data_dir.sh data/train_dnn_32k
+
+# Initialize a full GMM from the DNN posteriors and language recognition
+# features. This can be used both alone, as a UBM, or to initialize the
+# i-vector extractor in a DNN-based system.
+lid/init_full_ubm_from_dnn.sh --nj 40 --cmd "$train_cmd -l mem_free=6G,ram_free=6G" \
+  data/train_32k \
+  data/train_dnn_32k $nnet exp/full_ubm
+
+
+# Train an i-vector extractor based on the DNN-UBM.
+lid/train_ivector_extractor_dnn.sh --cmd "$train_cmd -l mem_free=80G,ram_free=80G" \
+				   --min-post 0.015 \
+				   --ivector-dim 600 --nj 1 --num-processes 1 --num-threads 8 \
```

Comment:
Can you remove the `--nj 1 --num-processes 1 --num-threads 8`? The default options should be OK, and more consistent with similar scripts.